builder/vsphere/driver/library.go

```diff
@@ -18,3 +21,44 @@ func (d *Driver) FindContentLibrary(name string) (*Library, error) {
 		driver:  d,
 	}, nil
 }
+
+func (d *Driver) FindContentLibraryItem(libraryId string, name string) (*library.Item, error) {
+	lm := library.NewManager(d.restClient)
+	items, err := lm.GetLibraryItems(d.ctx, libraryId)
+	if err != nil {
+		return nil, err
+	}
+	for _, item := range items {
+		if item.Name == name {
+			return &item, nil
+		}
+	}
+	return nil, nil
```

Comment:
should this return an error so that the caller does not panic in that case ?

Reply:
it has to be nil because not finding a library item is one of possible output here and it doesn't mean there's an error.

Reply:
got it 🙂